cardano-testnet/src/Testnet/Babbage.hs

```diff
@@ -335,7 +335,7 @@ babbageTestnet testnetOptions H.Conf {..} = do
     return $ PoolNode runtime key
 
   now <- H.noteShowIO DTC.getCurrentTime
-  deadline <- H.noteShow $ DTC.addUTCTime 90 now
+  deadline <- H.noteShow $ DTC.addUTCTime 110 now
```

Comment:
Why has this been increased? Was it too low?

Reply:
I suspected it was too low, but I'm thinking maybe it wasn't and my test failures are due to something else.  Will try bumping it back down again.

Reply:
I've verified I don't need this change.